### DISCUSSION TITLE
v4.0.1: Install specific solvers from PyPI

### DISCUSSION THREAD

**Discussion by stephane-caron** [qpsolvers/qpsolvers#249] [Announcements]
This minor release adds the ability to install specific solvers from PyPI, *e.g.* `pip install qpsolvers[clarabel,daqp,proxqp,scs]`.

Thanks to @sandeep026 for suggesting this in https://github.com/qpsolvers/qpsolvers/discussions/247 :+1: 

### Added

- Allow installation of a subset of QP solvers from PyPI

<hr /><em>This discussion was created from the release <a href='https://github.com/qpsolvers/qpsolvers/releases/tag/v4.0.1'>v4.0.1</a>.</em>

**Comment by sandeep026** [comment]
Thank you for adding the feature.
